entity-view/testsuite/src/test/java/com/blazebit/persistence/view/testsuite/correlation/general/model/DocumentCorrelationViewSubselect.java

```diff
@@ -32,39 +30,22 @@
  * @since 1.2.0
  */
 @EntityView(Document.class)
-public abstract class DocumentCorrelationViewSubselect implements DocumentCorrelationView {//extends DocumentCorrelationView {
```

Comment:
What removal? Since it is not needed any more, the comment should be removed and not left in there.